.expeditor/release.omnibus.yml

```diff
@@ -10,7 +10,6 @@ fips-platforms:
   - windows-*
 skip-artifactory-platforms:
   - freebsd-13-amd64
-windows-64-msystem: UCRT64
```

Comment:
just confirming that we intend to leave this out for now.

Reply:
Correct. We will need to put it back in shortly to get Ruby 3.1 support in